doc/book/connectors/__go.rst

```diff
@@ -52,7 +52,7 @@ Last update: January 2022
         -   12
 
     *   -   Static analysis
-        -   No
+        -   Yes
```

Comment:
We mention a tool for other connectors and it looks meaningful to do that here too.

Reply:
Ok.

```diff
--- a/doc/book/connectors/__go.rst
+++ b/doc/book/connectors/__go.rst
@@ -52,7 +52,7 @@ Last update: January 2022
         -   12
 
     *   -   Static analysis
-        -   Yes
+        -   golangci-lint, luacheck
         -   golint
         -   golangci-lint
 
```